src/Action/NullAction.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ApiPlatform\Core\Action;
+
+/**
+ * Empty action. Useful to trigger the kernel.view event without doing anything specific in the action
+ * (e.g. the POST action).
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+final class NullAction
+{
+    public function __invoke()
```

Comment:
Interesting, it doesn't throw an exception as highlighted by tests (so there is an error in the doc anyway). However maybe should we return another value than `null`, I honestly don't know.